src/Get.test.tsx

```diff
@@ -102,6 +101,54 @@ describe("Get", () => {
     });
   });
 
+  fdescribe("composing nested urls", () => {
+    it("should not compose if a absolute path is given", async () => {
+      nock("https://my-awesome-api.fake")
+        .get("/plop")
+        .reply(200);
+      nock("https://my-awesome-api.fake")
+        .get("/boom")
+        .reply(200);
+
+      const nestedGetChildren = jest.fn(() => <div />);
+      const firstGetChildren = jest.fn(() => <Get path="/boom">{nestedGetChildren}</Get>);
```

Comment:
You should use `render()` as every other tests, you can directly neast components in this `render` function.

`nock` throw if the endpoint is not called (I invite you to try and see by yourself :wink:) so you can use this behaviour as unit test (if the url is not called, the test failed)

Reply:
Is it something like this that you are suggesting?

```typescript
     const parentRoute = nock("https://my-awesome-api.fake")
        .get("/plop")
        .reply(200);
      const nestedRoute = nock("https://my-awesome-api.fake")
        .get("/plop/boom")
        .reply(200);

      render(
        <RestfulProvider base="https://my-awesome-api.fake">
          <Get path="plop">{() => <Get path="boom">{() => <div />}</Get>}</Get>
        </RestfulProvider>,
      );

      await wait(() => {
        parentRoute.done();
        nestedRoute.done();
      });
```

Reply:
It can works! never try with `nock.done()` but it's actually a very good idea 👍